Upload step for copy process completion added (#146)
mulesoft/docs-rpa: rpa-manager/modules/ROOT/pages/_partials/form-scheduling.adoc

+
The duration of the contained time interval and the start time are taken into account. The duration of the time interval is defined as the difference between the *From* and *To* times. The *From* time is the start time of the first execution. The start time of repetitions depends on the other set values.
+

> If you want to execute the repeats only with reference to the time of day, you should ensure that the start and end date are identical. If the interval specified  covers multiple days, a new multi-day interval begins with every repetition. This can lead to a situation whereby the scheduled time interval lasts longer than expected because the full number of days contained in the *From-To* interval is executed once again during the last repetition.

Reviewer: ```suggestion
To execute the repeats only with reference to the time of day, ensure that the start and end dates are identical. If the interval specified covers multiple days, a new multi-day interval begins with every repetition. This can lead to a situation in which the scheduled time interval lasts longer than expected because the full number of days contained in the *From-To* interval is executed once again during the last repetition.
```

Reviewer: Suggest: To execute the repeats referencing only the time of day

+
With schedules that are not all day, the actions in each repetition are executed for the duration of the interval specified in *From* and *To*.

*** *Repeat minutely*:
+

> If you select *Repeat minutely*, the planned action is executed every minute during the interval specified via *To* and *From*.

Reviewer: This is a preference - during the interval specified in the *From* and *To* fields.

+
You can select whether you want to repeat the action every minute or only every _n_-th minute. _n_ here stands for any integer number between 1 and 59.
*** *Repeat hourly*:
+
If you select *Repeat hourly*, the planned action is executed hourly during the interval specified via *To* and *From*. You can select whether you want to repeat the action every hour or only every _n_-th hour. _n_ here stands for any integer number between 1 and 23.
*** *Repeat daily*:
+
If you select *Repeat daily*, the planned action is executed daily during the interval specified via *To* and *From*.
+
You can select whether you want to repeat the action every day or only every _n_-th day. _n_ here stands for any integer number between 1 and 31.
*** *Repeat weekly*:
+

> If you select Repeat weekly, the planned action is executed weekly during the interval specified via *To* and *From*.

Reviewer: ```suggestion
If you select *Repeat weekly*, the planned action is executed weekly during the interval specified via *To* and *From*.
```

Reviewer: Bold *Repeat weekly* because it is a UI element. (You bolded Repeat daily. Same comment for similar occurrences.
